rpmbuild/copr_rpmbuild/automation/fedora_review.py

```diff
@@ -10,6 +10,7 @@
 
 import os
 import shutil
+from copr_common.helpers import chroot_to_branch
```

Comment:
+1 for moving the method

Reply:
This change is no longer needed because we won't use `chroot_to_branch` in `copr-rpmbuild` after all. But since you +1'd it, I left the commit there.